platform-sdk/docs/proposals/metric-labels/README.md

```diff
@@ -0,0 +1,175 @@
+# Metric labels
+
++author: @hendrikEbbers
++status: draft
+
+## Summary
+
+This proposal describes a new feature for the metrics module.
+The feature is called "metric labels" and allows to add labels to metrics.
+Labels are key-value pairs that can be used to filter and group metrics.
+Why this proposal uses the name "metric labels" (like prometheus does) other libraries (like micrometer) use the term tags.
+
+## Motivation
```

Comment:
I started using tags with this Helidon module I was writing. I actually don't know how it manifests itself in Grafana. I am interested to find out. I think for example I can create a metric "transactions-per-second" and then have tags for the actual transaction type, and this will somehow give me overall tps plus per-type tps?